pkg/services/object/put/distributed.go

```diff
@@ -165,7 +161,8 @@ func (t *distributedTarget) Close() (oid.ID, error) {
 			err = errIncompletePut{singleErr: fmt.Errorf("%w (last node error: %w)", errNotEnoughNodes{required: 1}, err)}
 		}
 	} else {
-		err = t.placementIterator.iterateNodesForObject(id, t.sendObject)
+		broadcast := tombOrLink || (!t.localOnly && typ == objectSDK.TypeLock) || len(t.obj.Children()) > 0
```

Comment:
Makes me wonder about `localOnly && tombOrLink`, but at least that's the way it was done previously.